content/settings/README.md

```diff
@@ -0,0 +1,20 @@
+# Content / Settings
```

Comment:
Please add any files of this folder to the gitignore file (except of the README)

Reply:
Done. The README files should already be excluded